app/views/admin/order_cycles/edit.html.haml

```diff
@@ -15,19 +15,46 @@
   = t :edit_order_cycle
 
 - ng_controller = @order_cycle.simple? ? 'AdminSimpleEditOrderCycleCtrl' : 'AdminEditOrderCycleCtrl'
+- has_scheduled_order = @order_cycle.schedules.exists?
```

Comment:
According to the issue, I think this should be checking for linked orders, not schedules. 

> When changing the date range of an OC **which has linked orders**

There is a DB field `orders.order_cycle_id`. So I think we can add a relationship to `order.rb`: `has_many :orders` (I'm not sure why this isn't there already).

Then it should be easy to check if `@order_cycle.orders.exists?`